### ISSUE TITLE
Owncloud: notes path found on server => not found

### ISSUE THREAD

**Issue by alenorcy** [pbek/QOwnNotes#1725]
Hello, 
I try to use QownNotes with my Owncloud server... but I get "notes path found on server" not found.

I saw this link (=> https://www.qownnotes.org/Knowledge-base/QOwnNotes-settings-are-reporting-a-problem-with-my-ownCloud-path) but "main ownCloud sync folder" parameter no longer exists

#### Expected behaviour

The owncloud connection seems good.

Apache log  :

X.X.X.X - - [02/May/2020:15:54:13 +0200] "GET / HTTP/1.1" 302 - "-" "Mozilla/5.0"
X.X.X.X - jean.testeur [02/May/2020:15:54:13 +0200] "GET /ocs/v1.php/cloud/capabilities HTTP/1.1" 200 2922 "-" "Mozilla/5.0 (X11; U; Linux i686; en-US; rv:1.9a3pre) Gecko/20070330"
X.X.X.X - jean.testeur [02/May/2020:15:54:13 +0200] "GET /index.php/apps/qownnotesapi/api/v1/note/app_info?format=json&notes_path=/Notes/ HTTP/1.1" 200 133 "-" "Mozilla/5.0 (X11; U; Linux i686; en-US; rv:1.9a3pre) Gecko/20070330"

And when, l try directly in a web browser : https://box-dev.mondomaine.fr/index.php/apps/qownnotesapi/api/v1/note/app_info?format=json&notes_path=/Notes/

I get a correct answer :  
{"versions_app":true,"trash_app":true,"versioning":true,"app_version":"19.1.0","server_version":"10.3.2.2","notes_path_exists":true}

#### Actual behaviour

Note folder path: /home/jean.testeur/Documents/Box-dev/Notes
Use note sub-folders : check
Path on your Nextcloud /owncloud server: Notes (I find it in the trees)

The last check failed :

Connection status
ok connection to web server
ok Nextcloud /ownCloud server detected
ok username and password correct
ok connection to QOwnNotesAPI
ok versions app enabled on server
ok trash app enabled on server
not found notes path found on server

The connection was made successfully!
Server version: 10.3.2.2
QOwnNotesAPI version: 19.1.0

#### Steps to reproduce

OS client : Ubuntu 18.04 LTS
QOwnNotes - version 20.5.1 - Launchpad PPA - build 709
Owncloud 10.3.2.2 (CentOS7) with QOwnNotesAPI version: 19.1.0

#### Output from the debug section in the settings dialog

QOwnNotes Debug Information
===========================

## General Info

**Current Date**: `sam. mai 2 15:56:14 2020`
**Version**: `20.5.1`
**Build date**: `May  1 2020`
**Build number**: `709`
**Platform**: `linux`
**Operating System**: `Ubuntu 18.04.4 LTS`
**Build architecture**: `x86_64`
**Current architecture**: `x86_64`
**Release**: `Launchpad PPA`
**Qt Version (build)**: `5.9.5`
**Qt Version (runtime)**: `5.9.5`
**Portable mode**: `no`
**Settings path / key**: `/home/jean.testeur/.config/PBE/QOwnNotes.conf`
**Application database path**: `/home/jean.testeur/.local/share/PBE/QOwnNotes/QOwnNotes.sqlite`
**Application arguments**: `QOwnNotes`
**Qt Debug**: `0`
**Locale (system)**: `fr_FR`
**Locale (interface)**: `en`
**Primary screen resolution**: `1680x1050`
**Screen resolution(s)**: `1680x1050, 1680x1050`
**Icon theme**: `ubuntu-mono-light`
**Notes in current note folder**: `27`
**Calendar items**: `0`
**Enabled scripts**: `0`

## Server Info

**serverUrl**: `https://box-dev.mondomaine.fr`
**appIsValid**: `yes`
**notesPathExists**: `not found`
**serverVersion**: `10.3.2.2`
**appVersion**: *empty*

## Spellchecking

**Enabled**: `true`
**Selected language**: `auto`
**Language codes**: `en_US, fr, fr_BE, fr_CA, fr_CH, fr_FR, fr_LU, fr_MC`
**Language names**: `American English (United States), français, français (Belgique), français canadien (Canada), français suisse (Suisse), français (France), français (Luxembourg), français (Monaco)`
**Application dictionaries path**: `/home/jean.testeur/.local/share/PBE/QOwnNotes/dicts`

## Note folders

**currentNoteFolderId**: `2`

### Note folder ``

**id**: `2`
**isCurrent**: `yes`
**activeTagId**: `-1`
**localPath**: `/home/jean.testeur/Documents/Box-dev/Notes`
**remotePath**: `Notes`
**cloudConnectionId**: `1`
**isShowSubfolders**: `yes`
**isUseGit**: `no`
**activeNoteSubFolder name**: *empty*
**database file**: `/home/jean.testeur/Documents/Box-dev/Notes/notes.sqlite`

### Cloud connection `Box-dev`

**id**: `1`
**isCurrent**: `yes`
**serverUrl**: `https://box-dev.mondomaine.fr`
**username**: `jean.testeur`

## Settings

...
**notesPath** (QString): `/home/jean.testeur/Documents/Box-dev`
...
**ownCloudInfo/appIsValid** (bool): `true`
**ownCloudInfo/connectionErrorMessage** (QString): `Erreur inconnue`
**ownCloudInfo/notesPathExistsText** (QString): `not found`
**ownCloudInfo/serverVersion** (QString): `10.3.2.2`
...

</details>

I find strange that "notesPath" parameter is "/home/jean.testeur/Documents/Box-dev" instead of "/home/jean.testeur/Documents/Box-dev/Notes" (like "localPath")

But when i set "/home/jean.testeur/Documents/Box-dev" in "Note folder path"  (and "Notes" or "<empty>" in the "Path on your Nextcloud /owncloud server") it's the same..

Thanks for your help

**Comment by pbek** [comment]
> I saw this link (=> https://www.qownnotes.org/Knowledge-base/QOwnNotes-settings-are-reporting-a-problem-with-my-ownCloud-path) but "main ownCloud sync folder" parameter no longer exists

Thank you, that article is really outdated. 😁 

Your settings look ok, did you sync your notes with your Nextcloud sync client to your server?
See: https://docs.qownnotes.org/en/latest/getting-started/concept/

**Comment by pbek** [comment]
It's indeed strange how your `notesPath` is set, it normally is overwritten by your note folder settings. Maybe your first note folder (id 1, you may have deleted it) had that set. Can you please try to create a new note folder setting (your current one will be cloned) and see if `notesPath` will be set to the same directory as your note folder.

**Comment by alenorcy** [comment]
I sync my notes with Owncloud sync client (2.5.1).

I create a new note folder..( and I deleted all the others) and now notesPath = localPath = "/home/jean.testeur/Documents/Box-dev/Notes"

but i alway get "notes path found on server" => not found

Thank you

**Comment by pbek** [comment]
1) Can you please try to create a new note folder setting (your current one will be cloned) and see if notesPath will be set to the same directory as your note folder.
2) And is there any useful log output if you enable the log panel (see Window / Panels) while you get the error message? Please make sure you also enable debug logging.

**Comment by alenorcy** [comment]
1. Yes, when i create a new note folder (it's cloned), now, "notesPath" it set to the same directory as my note folder ("/home/jean.testeur/Documents/Box-dev/Notes")

2. I have put log in debug. In the log panel, there is nothing special (no error)

When, I do "Connect To NC / oC Server" (in the Nextcloud / ownCloud settings, I always have "notes path found on server" with "not found" and at the same time I have this in the logs (/home/jean.testeur/.local/share/PBE/QOwnNotes/QOwnNotes.log) :

[mai 03 14:26:36] [debug]: Set proxy configuration to use NO proxy
[mai 03 14:26:37] [debug]: cloudConnection:  CloudConnection: <id>1 <name>"Box-dev" <serverUrl>"https://box-dev.mondomaine.fr" <username>"jean.testeur" <priority>1
[mai 03 14:26:37] [debug]: Reply from  ""
[mai 03 14:26:37] [debug]: Reply from main server url
[mai 03 14:26:37] [debug]: Reply from  "/ocs/v1.php"
[mai 03 14:26:37] [debug]: Reply from ownCloud test page
[mai 03 14:26:37] [debug]: Reply from  "/index.php/apps/qownnotesapi/api/v1/note/app_info"
[mai 03 14:26:37] [debug]: Reply from app info
[mai 03 14:26:37] [debug]: Reply from  "/ocs/v1.php/cloud/capabilities"
[mai 03 14:26:37] [debug]: Reply from capabilities page

===

why, as already indicated, when, l try directly in a web browser (after authentification) : https://box-dev.mondomaine.fr/index.php/apps/qownnotesapi/api/v1/note/app_info?format=json&notes_path=/Notes/ (it's what i found in apache log)
I get this : {"versions_app":true,"trash_app":true,"versioning":true,"app_version":"19.1.0","server_version":"10.3.2.2","notes_path_exists":true}

and and especially : "notes_path_exists":true, does it look good though?

Thank you for help

**Comment by pbek** [comment]
What does <https://box-dev.mondomaine.fr/apps/files/?dir=/Notes> show you? (I get a `box-dev.mondomaine.fr’s server IP address could not be found.`)

**Comment by pbek** [comment]
And do you have a screenshot where you get the error "not found"?

**Comment by alenorcy** [comment]
![Settings_242](https://user-images.githubusercontent.com/54445029/80923934-b92ba200-8d86-11ea-9370-9951c02311e5.jpg)
![Settings_243](https://user-images.githubusercontent.com/54445029/80923939-c052b000-8d86-11ea-8833-4d5205c0044b.jpg)

"https://box-dev.mondomaine.fr/apps/files/?dir=/Notes" refers to the Notes / directory of my owncloud account (with user jean.testeur)

It is the owncloud of my company, I voluntarily hide the real name and replaced by "mondomaine.fr". Besides, it's the preprod owncloud, it's not open.

Thank you.

**Comment by pbek** [comment]
What happens if you click on this button:

![image](https://user-images.githubusercontent.com/1798101/80936209-64ad1480-8dd0-11ea-963f-dab8d20d219e.png)

**Comment by pbek** [comment]
And there should be a log-output in the log panel...

![image](https://user-images.githubusercontent.com/1798101/80936366-2e23c980-8dd1-11ea-965b-6d046f53679f.png)

**Comment by alenorcy** [comment]
![Sélection_248](https://user-images.githubusercontent.com/54445029/80946689-deec9180-8dee-11ea-903c-b8570b999c45.jpg)

nothing in log panel :
![Sélection_245](https://user-images.githubusercontent.com/54445029/80946705-e744cc80-8dee-11ea-8c9e-c8d2ac64b96b.jpg)

yet I put the logs in debug :
![Sélection_246](https://user-images.githubusercontent.com/54445029/80946773-0d6a6c80-8def-11ea-9dd1-31b73b5d4eab.jpg)

In file "/home/jean.testeur/.local/share/PBE/QOwnNotes/QOwnNotes.log" (when i get the error notes path found on server "not found") :

[mai 04 09:55:06] [debug]: Set proxy configuration to use NO proxy
[mai 04 09:55:07] [debug]: cloudConnection:  CloudConnection: <id>1 <name>"box-dev" <serverUrl>"https://box-dev.mondomaine.fr" <username>"jean.testeur" <priority>1
[mai 04 09:55:07] [debug]: Reply from  ""
[mai 04 09:55:07] [debug]: Reply from main server url
[mai 04 09:55:07] [debug]: Reply from  "/ocs/v1.php"
[mai 04 09:55:07] [debug]: Reply from ownCloud test page
[mai 04 09:55:07] [debug]: Reply from  "/ocs/v1.php/cloud/capabilities"
[mai 04 09:55:07] [debug]: Reply from capabilities page
[mai 04 09:55:07] [debug]: Reply from  "/index.php/apps/qownnotesapi/api/v1/note/app_info"
[mai 04 09:55:07] [debug]: Reply from app info

**Comment by pbek** [comment]
> nothing in log panel

It's easier to turn on debug logs in that panel (right mouse click on it to get the menu)

> [mai 04 09:55:07] [debug]: Reply from capabilities page

It seems like that QOwnNotesAPI on your ownCloud server reports that the note folder is not found, I'll add some more debug info for you in the next release.

**Comment by pbek** [comment]
The `notes_path_exists` value should be set here:

https://github.com/pbek/qownnotesapi/blob/d3cab4bd7c6961a7b8bb28450d0653e600a99934/controller/noteapicontroller.php#L107-L130

**Comment by alenorcy** [comment]
ok, thank you for the right mouse click on log panel, sorry.

"notes_path_exists" seems yet OK..

 https://box-dev.mondomaine.fr/index.php/apps/qownnotesapi/api/v1/note/app_info?format=json&notes_path=/Notes/ 
=> {"versions_app":true,"trash_app":true,"versioning":true,"app_version":"19.1.0","server_version":"10.3.2.2","notes_path_exists":**true**}


 https://box-dev.mondomaine.fr/index.php/apps/qownnotesapi/api/v1/note/app_info?format=json&notes_path=/Whatever-other/ 
=> {"versions_app":true,"trash_app":true,"versioning":true,"app_version":"19.1.0","server_version":"10.3.2.2","notes_path_exists":**false**}

Anyway, I don't know what else to do :-)

Thank you

**Comment by pbek** [comment]
## 20.5.3
- added more debug output in case note folder was not found on server

**Comment by pbek** [comment]
There now is a new release, could you please test it and report the output in the log panel while you are opening the settings dialog?

**Comment by alenorcy** [comment]
**with 20.5.3 :**

> [22:00:40] [debug] Set proxy configuration to use NO proxy
> [22:00:40] [debug] cloudConnection: CloudConnection: <id>1 <name>"box-dev" <serverUrl>"https://box-dev.mondomaine.fr" <username>"jean.testeur" <priority>1
> [22:00:41] [debug] Reply from ""
> [22:00:41] [debug] Reply from main server url
> [22:00:41] [debug] Reply from "/ocs/v1.php/cloud/capabilities"
> [22:00:41] [debug] Reply from capabilities page
> [22:00:41] [debug] Reply from "/index.php/apps/qownnotesapi/api/v1/note/app_info"
> [22:00:41] [debug] Reply from app info
> [22:00:41] [debug] checkAppInfo - 'data': "[{\"versions_app\":true,\"trash_app\":true,\"versioning\":true,\"app_version\":\"19.1.0\",\"server_version\":\"10.3.2.2\",\"notes_path_exists\":false}]"
> [22:00:41] [debug] Reply from "/ocs/v1.php"

Shouldn't it be "/index.php/apps/qownnotesapi/api/v1/note/app_info?notes_path=/Notes"  instead of "/index.php/apps/qownnotesapi/api/v1/note/app_info" ? (according my "remotePath")

Because in a browser for "https://box-dev.mondomaine.fr/index.php/apps/qownnotesapi/api/v1/note/app_info?notes_path=/Notes" => "notes_path_exists":true
while "https://box-dev.mondomaine.fr/index.php/apps/qownnotesapi/api/v1/note/app_info" => "notes_path_exists":false

**Comment by pbek** [comment]
> Shouldn't it be "/index.php/apps/qownnotesapi/api/v1/note/app_info?notes_path=/Notes" instead of "/index.php/apps/qownnotesapi/api/v1/note/app_info" ? (according my "remotePath")

"Get parameters" are not shown in the reply.

Do you have access to your server logs? What requests did you get on `[22:00:41]`?

**Comment by alenorcy** [comment]
actually the parameter is well returned .. here are the log

Apache :
[apache_access_log.txt](https://github.com/pbek/QOwnNotes/files/4579473/apache_access_log.txt)

Owncloud :
[owncloud_log.txt](https://github.com/pbek/QOwnNotes/files/4579474/owncloud_log.txt)

**Comment by pbek** [comment]
Everything seems fine. I really don't understand why your server returns `"notes_path_exists":false` when you make the app_info request in QOwnNotes. Maybe some special permissions that are set on your server for your files or the `Notes` directory?

What happens if you share a note (*Note* menu)? Does that work?

**Comment by alenorcy** [comment]
I have no special permissions on my remote "Notes" directory. Everything is like the others in the tree (owner and group => apache). Besides, it is not linked to my test account, because I asked someone on my team to test and he has the same problem... I don't see either ..

well, note sharing works fine !

**Comment by pbek** [comment]
So I think it's really only a problem of these few lines of code on your server:

https://github.com/pbek/qownnotesapi/blob/d3cab4bd7c6961a7b8bb28450d0653e600a99934/controller/noteapicontroller.php#L111-L120

And that's hard to debug without having a ownCloud account on your server and access to that file. 😬

**Comment by pbek** [comment]
If you want you can add these two lines after https://github.com/pbek/qownnotesapi/blob/d3cab4bd7c6961a7b8bb28450d0653e600a99934/controller/noteapicontroller.php#L128 on your server in `apps/qownnotesapi/controller/noteapicontroller.php`:

```
            "debug_user" => $this->user,
            "debug_notes_path" => $notesPath,
```

...and then send me the debug log again when you open the QON settings dialog.

**Comment by alenorcy** [comment]
here is what i get after adding the 2 lines:

> [12:03:15] [debug] cloudConnection: CloudConnection: <id>1 <name>"box-dev" <serverUrl>"https://box-dev.mondomaine.fr" <username>"jean.testeur" <priority>1
> [12:03:15] [debug] Reply from ""
> [12:03:15] [debug] Reply from main server url
> [12:03:15] [debug] Reply from "/ocs/v1.php"
> [12:03:15] [debug] Reply from ownCloud test page
> [12:03:15] [debug] Reply from "/index.php/apps/qownnotesapi/api/v1/note/app_info"
> [12:03:15] [debug] Reply from app info
> [12:03:15] [debug] checkAppInfo - 'data': "[{\"versions_app\":true,\"trash_app\":true,\"versioning\":true,\"app_version\":\"19.1.0\",\"server_version\":\"10.3.2.2\",\"notes_path_exists\":false,\"debug_user\":null,\"debug_notes_path\":\"\\/files\\/Notes\\/\"}]"
> [12:03:15] [debug] Reply from "/ocs/v1.php/cloud/capabilities"
> [12:03:15] [debug] Reply from capabilities page

**Comment by pbek** [comment]
The user isn't set on your server. Very strange. Can you try to replace the `__construct` function with this:

```php
    /**
     * @param string $AppName
     * @param IUserManager $userManager
     * @param IRequest $request
     */
    public function __construct($AppName,
                                IUserManager $userManager,
                                IRequest $request) {
        $this->user = $userManager->getUser()->getUID();
        parent::__construct($AppName, $request);
    }
```

**Comment by alenorcy** [comment]
i get now (with the new  __construct function) a warning : 

> [22:59:07] [debug] cloudConnection: CloudConnection: <id>1 <name>"box-dev" <serverUrl>"https://box-dev.mondomaine.fr" <username>"jean.testeur" <priority>1
> [22:59:07] [debug] Reply from ""
> [22:59:07] [debug] Reply from main server url
> [22:59:07] [debug] Reply from "/index.php/apps/qownnotesapi/api/v1/note/app_info"
> [22:59:07] [warning] "QNetworkReply error 401 from url https://box-dev.mondomaine.fr/index.php/apps/qownnotesapi/api/v1/note/app_info?format=json&notes_path=/Notes/: " "Error transferring https://box-dev.mondomaine.fr/index.php/apps/qownnotesapi/api/v1/note/app_info?format=json&notes_path=/Notes/ - server replied: Internal Server Error"
> [22:59:07] [debug] Reply from app info
> [22:59:07] [debug] Reply from "/ocs/v1.php"
> [22:59:07] [debug] Reply from ownCloud test page
> [22:59:07] [debug] Reply from "/ocs/v1.php/cloud/capabilities"
> [22:59:07] [debug] Reply from capabilities page

![Sélection_253](https://user-images.githubusercontent.com/54445029/81228254-12e1d580-8fee-11ea-8885-7c12de7c92ad.jpg)

Thank you

**Comment by pbek** [comment]
Any log output on your server?

**Comment by alenorcy** [comment]
I had to add "use \OCP\IUserManager;"
But he doesn't like "$userManager->getUser()->getUID();"

W.X.Y.Z - jean.testeur [08/May/2020:19:26:34 +0200] "GET /index.php/apps/qownnotesapi/api/v1/note/app_info?format=json&notes_path=/Notes/ HTTP/1.1" 500 9794 "-" "Mozilla/5.0 (X11; U; Linux i686; en-US; rv:1.9a3pre) Gecko/20070330"

I try with IUserSession  (with use \OCP\IUserSession)

>     /**
>      * @param string $AppName
>      * @param IUserSession $userSession  
>      * @param IRequest $request
>      */
>     public function __construct($AppName,
>                                 IUserSession $userSession,
>                                 IRequest $request) {
>         $this->user = $userSession->getUser();
>         parent::__construct($AppName, $request);
>     }

No Apache error 500.. but we find the same initial problem :
> [19:16:22] [debug] checkAppInfo - 'data': "[{\"versions_app\":true,\"trash_app\":true,\"versioning\":true,\"app_version\":\"19.1.0\",\"server_version\":\"10.3.2.2\",\"notes_path_exists\":false,\"debug_user\":null,\"debug_notes_path\":\"\\/files\\/Notes\\/\"}]"

In __construct when i "force" (of course it works)
$this->user = "jean.testeur";

You can see.. it's fake..
![Sélection_257](https://user-images.githubusercontent.com/54445029/81431334-42fdb580-9161-11ea-838c-68c2df176ea7.jpg)

**Comment by pbek** [comment]
Thank you for testing, I really have no clue why your server doesn't provide the user id. :grimacing:

**Comment by alenorcy** [comment]
Hello,

I just did a new clean owncloud installation in version 10.3.2 (identical to the one I had) => the problem is still there! suddenly, I installed the latest version (10.4.2) => same

I installed them with ansible (https://github.com/owncloud-ansible/owncloud)

> [14:48:11] [debug] cloudConnection: CloudConnection: <id>2 <name>"owncloud" <serverUrl>"http://owncloud.mondomaine.fr" <username>"test" <priority>0
> [14:48:11] [debug] Reply from ""
> [14:48:11] [debug] Reply from main server url
> [14:48:11] [debug] Reply from "/ocs/v1.php"
> [14:48:11] [debug] Reply from ownCloud test page
> [14:48:11] [debug] Reply from "/ocs/v1.php/cloud/capabilities"
> [14:48:11] [debug] Reply from capabilities page
> [14:48:11] [debug] Reply from "/index.php/apps/qownnotesapi/api/v1/note/app_info"
> [14:48:11] [debug] Reply from app info
> [14:48:11] [debug] checkAppInfo - 'data': "[{\"versions_app\":true,\"trash_app\":true,\"versioning\":true,\"app_version\":\"19.1.0\",\"server_version\":\"10.4.1.3\",\"notes_path_exists\":false}]"

![Sélection_259](https://user-images.githubusercontent.com/54445029/81816268-62af2800-952b-11ea-870e-8084b766a611.jpg)

Thank you

**Comment by pbek** [comment]
Thank you for investigating. It would be pretty bad if ownCloud would start to diverge from their own API and generally don't set `$userId` in the `ApiController` of extensions. I really don't want to maintain different versions of QOwnNotesAPI for Nextcloud and ownCloud... In case you find something about the `$userId` issue in ownCloud please let me know.

**Comment by pbek** [comment]
I will close this issue until there is more information.

**Comment by CraftingGamerTom** [comment]
Thanks @pbek . It appears the syncing issue was related to a misconfiguration of an anti-malware app. I could not see new versions and I saw the error finding the notes folder and associated blame too quickly. I was able to resolve it by fixing the anti-virus configuration. I still have the issue above (cant find notes) but I'm not sure what the bug is actually affecting - I can access my notes trash and note version history.

**Comment by pbek** [comment]
>  I still have the issue above (cant find notes) but I'm not sure what the bug is actually affecting - I can access my notes trash and note version history.

If you can access the version history of a note than QOwnNotesAPI is working correctly.
Are there any log-output in QON and your oC server?

**Comment by pbek** [comment]
I went through the ordeal of setting up an ownCloud server. It seems ownCloud broke the user fetching somehow. The user the qownnotesapi app receives in `null`. I tried several other ways to fetch the current user (via `IUserSession` or `$UserId`), the user was always `null`.

I compared the way our user is fetched by the way ownCloud notes does it and they are the same:
https://github.com/owncloud/notes/blob/e49aa04f078d9ce8ea7f3c5fc6522aa403edf434/controller/notescontroller.php#L49

**Comment by pbek** [comment]
For some strange reason I was still able to fetch the trash and the versions of the current note (they both depend on the user name). @alenorcy, @CraftingGamerTom, @guilegatan, can you confirm that those two are working?

**Comment by pbek** [comment]
## QOwnNotesAPI 21.3.0
- added a workaround for the failing username detection in ownCloud 10.3+

@alenorcy, @CraftingGamerTom, @guilegatan, there now is a new ## QOwnNotesAPI release for ownCloud, could you please test it and report if it works for you?

**Comment by alenorcy** [comment]
hi,
it's ok for me now, thanks !

![Selection_130](https://user-images.githubusercontent.com/54445029/110946551-4a708b00-833f-11eb-9f01-a72aa283bd2c.png)

**Comment by CraftingGamerTom** [comment]
Hey @pbek , 
Sorry for my delayed response. I have logs if you'd like them still but I see you made progress without them. Thanks for diving into this issue! 

After gathering logs I updated the QOwnNotesAPI app (_cleared cache for the update to show as available if anyone else doesn't see it right away_) and the notes path is being found (same results as the screenshot above)

Much appreciated! :)

**Comment by pbek** [comment]
Great. Thank you for testing 😁

**Comment by yunus776** [comment]
#edo-signature img {max-width: 90%}

**Comment by guilegatan** [comment]
This also (partially) solved my issue, ticket #2063 (added screenshots in that ticket)

Everything is now green in the settings (the notes folder is found again), but I still geht the "CSRF check failed" error on clicking ok. Therefore I still keep the API disabled for now.

**Comment by pbek** [comment]
Then let's talk about that in #2063.
